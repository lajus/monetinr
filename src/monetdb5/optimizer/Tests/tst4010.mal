# testing the library for unsafe instructions
# bat C can still be shared with B
optimizer.setDebug();
optimizer.commonTerms();
optimizer.aliases();
b:= bat.new(:oid,:int);
c:= bat.new(:oid,:int);
bat.append(b,1);

# in this case D should be left alone
d:= bat.new(:oid,:int);
bat.append(d,2);

