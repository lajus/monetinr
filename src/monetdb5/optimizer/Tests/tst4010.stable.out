stdout of test 'tst4010` in directory 'monetdb5/optimizer` itself:


# 23:52:52 >  
# 23:52:52 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=39352 --set sql_port=45171 --set xquery_port=52900 --set monet_prompt= --trace --dbname=mTests_src_optimizer  tst4010.mal </dev/null
# 23:52:52 >  

# MonetDB server v5.9.0, based on kernel v1.27.0
# Serving database 'mTests_src_optimizer', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008- MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://eir.ins.cwi.nl:32348/
function user.main():void;
# testing the library for unsafe instructions 
# bat C can still be shared with B 
    b := bat.new(:oid,:int);
    c := bat.new(:oid,:int);
    bat.append(b,1);
# in this case D should be left alone 
    d := bat.new(:oid,:int);
    bat.append(d,2);
end main;
#optimizer.commonTerms actions= 0 time=12 usec 
#optimizer.aliases    actions= 0 time=4 usec 

# 23:52:52 >  
# 23:52:52 >  Done.
# 23:52:52 >  

